.envExample

```diff
@@ -1,2 +0,0 @@
-DATABASE_URL=postgres://username:mypassword@localhost:5432/dataBaseName
-TOKEN_SECRET=
```

Comment:
can we keep this file?

Reply:
yes I will revert it

Reply:
I will revert this in a separated pull request , currently I'm facing this problem with Git that I can't resolve and it takes a lot of time...
`To https://github.com/WebAhead-cs2/caps-look-api.git
 ! [rejected]        HEAD -> features/Create_validations_and_response_messages_for_project_controller (non-fast-forward)
error: failed to push some refs to 'https://github.com/WebAhead-cs2/caps-look-api.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.`
So can you please merge this and I'll upload it again ...

Reply:
@tawfiknasser , can you merge this please ?

Reply:
This should solve your problem: `git pull origin features/Create_validations_and_response_messages_for_project_controller --rebase --autostash`

How ever I will merge it...